datastream/dataset.py

```diff
@@ -87,6 +87,17 @@ def from_dataframe(dataframe: pd.DataFrame) -> Dataset[pd.Series]:
             get_item=lambda df, index: df.iloc[index],
         )
 
+    def new_columns(self, **kwargs):
+        '''
+        Appends new column(s) to ``self.dataframe`` using
```

Comment:
Have you looked at the built documentation?

Maybe this would look nice?
```
:attr:`~.MyClass.my_prop`
```
or this
```
:attr:`.MyClass.my_prop`
```

Reply:
Ah, didn't know about those, I can change to that :slightly_smiling_face: